assets/js/frontend/give-donations.js

```diff
@@ -98,6 +98,8 @@ Give = {
 					} else if (( args.precision + 1 ) > decimal_amount.length) {
 						decimal_amount = ( decimal_amount + '000000000' ).substr(0, args.precision + 1);
 					}
+				}else {
```

Comment:
@ravinderk, Need to correct formatting here.

Reply:
@jaydeeprami Fixed

Reply:
Thanks @ravinderk